src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/PrisonOffenderManager.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.digital.delius.jpa.standard.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+
+@EqualsAndHashCode(of = "prisonOffenderManagerId")
+@ToString(exclude = {"team","staff","probationArea", "responsibleOfficer" ,"managedOffender"})
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@Entity
+@Table(name = "PRISON_OFFENDER_MANAGER")
+public class PrisonOffenderManager {
+
+    @Id
+    @Column(name = "PRISON_OFFENDER_MANAGER_ID")
+    private Long prisonOffenderManagerId;
+
+    @JoinColumn(name = "ALLOCATION_TEAM_ID")
+    @OneToOne
+    private Team team;
+
+    @JoinColumn(name = "ALLOCATION_STAFF_ID")
+    @OneToOne
+    private Staff staff;
+
+    @Column(name = "OFFENDER_ID")
+    private Long offenderId;
+
+    @Column(name = "SOFT_DELETED")
+    private Long softDeleted;
+
+    @JoinColumn(name = "PROBATION_AREA_ID")
+    @OneToOne
+    private ProbationArea probationArea;
+
+    @Column(name = "ACTIVE_FLAG")
+    private Long activeFlag;
+
+    @Column(name = "ALLOCATION_DATE")
+    private Timestamp allocationDate;
```

Comment:
I think the pattern is to use LocaDate and LocalDateTime in our entities

Reply:
Fair point - I will look to change that shortly. The other APIs (elite2 etc) all use LocalDateTime and LocalDate, so it does stand out as an oddity.